its/ruling/src/test/resources/commons-beanutils/java-S2701.json

```diff
@@ -1,8 +1,12 @@
 {
+'commons-beanutils:commons-beanutils:src/test/java/org/apache/commons/beanutils2/BeanToPropertyValueTransformerTestCase.java':[
+186,
+244,
+],
 'commons-beanutils:commons-beanutils:src/test/java/org/apache/commons/beanutils2/BeanUtilsTestCase.java':[
-210,
+209,
```

Comment:
The issue location slightly moved (and potentially the message aswell), from the literal to the method name, with the literal being the secondary location.
@m-g-sonar I'm afraid that this will cause side effects, re-open issues for example, what do you think?

Reply:
I would consider this as an improvement of the rule anyway, so I think it's perfectly fine.